### PR TITLE
Add clusters for Matter 1.2 device types

### PR DESCRIPTION
**Pull request by ctowns**
Adds new Matter 1.2 clusters for these types directly to the driver. Cluster definitions have been stripped down to just the essentials that are used in the drivers.

### CODE REVIEW

File: drivers/SmartThings/matter-sensor/src/AirQuality/server/attributes/AcceptedCommandList.lua [hunk 1]
```diff
@@ -0,0 +1,75 @@
+local cluster_base = require "st.matter.cluster_base"
```
Comment (tpmanley) [inline drivers/SmartThings/matter-sensor/src/AirQuality/server/attributes/AcceptedCommandList.lua:1]: Do we need this file? Air Quality doesn't have any commands?

File: drivers/SmartThings/matter-sensor/src/AirQuality/server/attributes/EventList.lua [hunk 2]
```diff
@@ -0,0 +1,75 @@
+local cluster_base = require "st.matter.cluster_base"
```
Comment (tpmanley) [inline drivers/SmartThings/matter-sensor/src/AirQuality/server/attributes/EventList.lua:1]: Do we need this file? Air Quality doesn't have any events

Comment (ctowns) [reply drivers/SmartThings/matter-sensor/src/AirQuality/server/attributes/EventList.lua:1]: Removed in 2df4f7a8fa7e0847b68a82f2e06417ac63f957cf!

File: drivers/SmartThings/matter-sensor/src/AirQuality/server/attributes/AcceptedCommandList.lua (hunk 1 above)
Comment (ctowns) [reply drivers/SmartThings/matter-sensor/src/AirQuality/server/attributes/AcceptedCommandList.lua:1]: Removed in 2df4f7a8fa7e0847b68a82f2e06417ac63f957cf!

**Comment by ctowns** [comment]
Added remaining clusters for Matter 1.2 device types. So far I have tested the matter-appliance and matter-sensor drivers and they both seem to be working on a hub without Matter 1.2 clusters (i.e with old lua libs). I'll test the other drivers and report the results here as well.

**Comment by ctowns** [comment]
Updated PR that was merged here: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1390

**Pull request closed without merging**